css/style.css

```diff
@@ -413,3 +407,118 @@ input:placeholder-shown{
     width: 25%
   }
 
+/* Stretch goals
+   Change the colour of the radio buttons and checkbox to the matching ORANGE colour
+*/
+
+/* First step: hide the unstyleable radio */
+ 
+label > input[type="radio"] {
+  display: none;
+ }
+  
+ /* make our own radio button
+ Making an empty circle is easy with CSS, just put a border around a square element and give it a border radius of 50%.
+ ... we'll put this on the ::before pseudo-element to avoid extra markup. */
+  
+ label > input[type="radio"] + *::before {
+  content: "";
+  display: inline-block;
+  vertical-align: bottom;
+  width: 1rem;
+  height: 1rem;
+  margin-right: 0.3rem;
+  border-radius: 50%;
+  border-style: solid;
+  border-width: 0.1rem;
+  border-color: gray;
+ }
+  
+ /* Last step: make our radio change when checked ... using :checked we can style our radio button to make it appear different when selected. */
+ /* use a radial-gradient to fill only a portion of the circle in. */
+  
+ label > input[type="radio"]:checked + *::before {
+  background: radial-gradient(#ff6200 0%, #ff6200 40%, transparent 50%, transparent);
+  border-color:#ff6200;
+ }
+ /* However do NOT colour the text!
+  
+ label > input[type="radio"]:checked + * {
+  color: #ff6200;
+ }
+  
+ */
+
+ 
+ /* First step: hide the unstyleable checkbox */
```

Comment:
I like how your comments explain what you are doing, makes following along very easy